community-edition/get_ip/index.js

```diff
@@ -4,13 +4,21 @@ const { spawnSync } = require("node:child_process");
 const config = utils.readConfig();
 const { stdout } = spawnSync(
   "kubectl",
-  ["-n", config.Namespace, "get", "svc", "lb", "-o", "json"],
+  ["get", "svc", "--field-selector", "spec.type=LoadBalancer", "-A", "-o", "json"],
   { stdio: ["pipe", "pipe", "inherit"] }
 );
 const output = JSON.parse(stdout);
-const ipAddr = output.status.loadBalancer?.ingress?.[0]?.ip;
-if (ipAddr) {
-  console.log("load balancer external IP address:", ipAddr);
-} else {
-  console.log("load balancer not running yet:", output.status.loadBalancer);
+if (output.items.length === 0) {
+  console.warn("can't determine external IP address: no load balancers in cluster");
+  process.exit(1);
+}
+for (const item of output.items) {
+  const name = item.metadata.name;
+  const namespace = item.metadata.namespace;
+  const ipAddr = item.status.loadBalancer?.ingress?.[0]?.ip;
+  if (ipAddr) {
+    console.log(`load balancer “${name}” in namespace “${namespace}” external IP address: ${ipAddr}`);
+  } else {
+    console.log(`load balancer “${name}” in namespace “${namespace}” not running yet:`, output.status.loadBalancer);
```

Comment:
I believe it should be `item.status.loadBalancer` now, instead of `output.status.loadBalancer`

Reply:
Done.